H1ml.fm1

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Page Title</title>
+  <style>
+    body {background-color: powderblue;}
+    h1 {color: black;}
+    p {color: black;}
+  </style>
+</head>  
+<body>
+
+<h1>Need to mix up your music taste?</h1>
+<img src="m" alt="music-background" width="250" height="200">
+
+<h2>Try our music generator and experience a new vibe for any enviorment.</h2>
+
+<button type="button"
```

Comment:
I am a little confused on what all of this is. I don't think you need to declare button type as button as well as we have a function loadForm() that can be called onclick to load the second html